### PR TITLE
Moved Grasshopper Components

### PR DESCRIPTION
**Pull request by MaciejDengusiak**
### Issues addressed by this PR

Fixes #

### CODE REVIEW

File: Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                return;
+            }
+            double relativeHumidity = double.NaN;
+            if (!dataAccess.GetData(index, ref relativeHumidity) || double.IsNaN(relativeHumidity))
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+
+            IsotermicHumidificationProcess isotermicHumidificationProcess = Core.Mollier.Create.IsotermicHumidificationProcess_ByHumidityRatioDifference(mollierPoint, relativeHumidity);
```
Comment (ZiolkowskiJakub) [inline Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity.cs:90]: Change to RelativeHumidity Create Method

File: Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateSteamHumidificationProcessByRelativeHumidity.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                return;
+            }
+            double relativeHumidity = double.NaN;
+            if (!dataAccess.GetData(index, ref relativeHumidity) || double.IsNaN(relativeHumidity))
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+
+            SteamHumidificationProcess steamHumidificationProcess = Core.Mollier.Create.SteamHumidificationProcess_ByHumidityRatioDifference(mollierPoint, relativeHumidity);
```
Comment (ZiolkowskiJakub) [inline Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateSteamHumidificationProcessByRelativeHumidity.cs:90]: Change to RelativeHumidity Create Method

**Review by ZiolkowskiJakub — CHANGES_REQUESTED** [review]

File: Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateCoolingProcessByMedium.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        protected override System.Drawing.Bitmap Icon => Resources.SAM_Small;
+
+        public override GH_Exposure Exposure => GH_Exposure.primary;
+
+        protected override GH_SAMParam[] Inputs
+        {
+            get
+            {
+                List<GH_SAMParam> result = new List<GH_SAMParam>();
+                result.Add(new GH_SAMParam(new GooMollierPointParam() { Name = "_start", NickName = "_start", Description = "Start Point for MollierProcess", Access = GH_ParamAccess.item }, ParamVisibility.Binding));
```
[the remark this answers is not in the document]
Comment (MaciejDengusiak) [reply Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateCoolingProcessByMedium.cs:33]: done

File: Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity.cs (hunk 1 above)
Comment (MaciejDengusiak) [reply Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateIsotermicHumidificationProcessByRelativeHumidity.cs:90]: done

File: Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateCoolingProcessByMedium.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                return;
+            }
+            double medium = double.NaN;
+            if (!dataAccess.GetData(index, ref medium) || double.IsNaN(medium))
+            {
+                AddRuntimeMessage(GH_RuntimeMessageLevel.Error, "Invalid data");
+                return;
+            }
+
+            CoolingProcess coolingProcess = Core.Mollier.Create.CoolingProcess(mollierPoint, medium);
```
[the remark this answers is not in the document]
Comment (MaciejDengusiak) [reply Grasshopper/SAM.Core.Grasshopper.Mollier/Component/SAMMollierCreateCoolingProcessByMedium.cs]: done

**Review by ZiolkowskiJakub — APPROVED** [review]

**Pull request merged**